bundle/Resources/views/themes/ngadmin/content/location_view.html.twig

```diff
@@ -26,7 +26,13 @@
                     </svg>
                 </a>
                 <a href="{{ url }}">{{ content_type.name }}</a>
+
+                {% set language_names = [] %}
```

Comment:
Why is the logic within span if we are rendering this outside of the span? :)

Reply:
Moved